src/modules/ModuleRunner.ts

```diff
@@ -29,6 +33,13 @@ import "./ModuleComponents";
 export class ModuleRunner {
     public static readonly instance = new ModuleRunner();
 
+    public className: string = ModuleRunner.name;
+
+    public extensions: AllExtensions = {
+        cryptoSetup: new DefaultCryptoSetupExtensions(),
+        experimental: new DefaultExperimentalExtensions(),
+    };
```

Comment:
Please can you make sure that any new or updated `public` or `export`ed symbols have TSDoc comments explaining what they are, and how they should be used. (Historically, we weren't very good at this but it is now a point on the PR checklist.)